obstacle_range: 2.5
raytrace_range: 3.0
footprint: [[-0.2, -0.15], [-0.2, 0.15], [0.2, 0.15], [0.2, -0.15]]
inflation_radius: 0.3
cost_scaling_factor: 3.0

observation_sources: scan

scan:
  sensor_frame: robot1_base_link
  data_type: LaserScan
  topic: /robot1/scan
  marking: true
  clearing: true
  min_obstacle_height: 0.0
  max_obstacle_height: 2.0
  obstacle_range: 2.5
  raytrace_range: 3.0